### ISSUE TITLE
Activate only by certains instruments

### ISSUE THREAD

**Issue by ThomasVermes** [jangari/REDCap_Requested_and_Required_fields#3]
It would be nice if we could choose to activate the EM only for certains instruments

**Comment by jangari** [comment]
@ThomasVermes Could you not just omit the tags on those instruments? I suppose this might compel you to specifically annotate `@REQUIRED` rather than using the required metadata option.

Anyway, I'll think about that.

**Comment by ThomasVermes** [comment]
that is true, that could be a workaround (or actually how you meant things to be =)
but at the moment the @required tag doesn't work to me
(see https://redcap.vumc.org/community/post.php?id=233354&comment=237556 )

sorry i'm mixing up issues and platforms!

**Comment by jangari** [comment]
That issue you raised turned out to be very simple (see #4), and I have now fixed it in v1.1.1. Maybe this will alleviate your need for instrument targeting?

**Comment by ThomasVermes** [comment]
eheh thanks a lot!
yes, then a would put this only under nice to have ;)
